packages/devtools_app/lib/src/screens/performance/performance_controller.dart

```diff
@@ -773,12 +811,14 @@ class PerformanceController extends DisposableController
 
     // Process trace events [processTraceEventsHelper] and time the operation
     // for analytics.
-    await ga.timeAsync(
-      analytics_constants.performance,
-      analytics_constants.traceEventProcessingTime,
-      asyncOperation: processTraceEventsHelper,
-      screenMetricsProvider: () => PerformanceScreenMetrics(
-        traceEventCount: processingTraceCount,
+    unawaited(
```

Comment:
why do we no longer want to await `processTraceEventsHelper`?

Reply:
the thing that is unawaited is the ga.timeAsync call.
I don't think we should slow down the UI waiting for the analytics event to complete.

Reply:
if we don't await the call to `ga.timeAsync`, then we inherently will not await the `asyncOperation` `processTraceEventsHelper`, right? We need to await `processTraceEventsHelper`, which is called in the body of the `timeAsync` method.

Reply:
dropped this from the PR.